docs/index.md

```diff
@@ -1,243 +1,163 @@
-  "scopes": [
-    "text.html.vue"
-  ],
-  "syntaxes": [
-    // For ST3 builds < 3153
-    "Packages/Vue Syntax Highlight/vue.tmLanguage"
-    // For ST3 builds >= 3153
-    // "Packages/Vue Syntax Highlight/Vue Component.sublime-syntax"
-  ]
+```jsonc
+// LSP.sublime-settings -- User
+{
+  "clients": {
+    "client_name": {
+      "command": ["pyls"],          // required
+      "scopes": ["source.python"],  // required
+      "syntaxes": ["Packages/Python/Python.sublime-syntax"], // required
+      "languageId": "python",       // required
+      "settings": { },              // optional
+      "initializationOptions": { }, // optional
+      "env": { },                   // optional
+      "enabled": false              // defaults to false, but needs to be true for the server to start
+    }
+  }
 }
 ```
 
-Be sure to install "Vue Syntax Highlight" from Package Control.
-
-### Python<a name="python"></a>
-
-`pip install python-language-server`
-
-See: [github:palantir/python-language-server](https://github.com/palantir/python-language-server)
-
-Alternatively, Microsoft's python language server (using .NET Core runtime)
-
-[Instructions here](https://github.com/Microsoft/python-language-server/blob/master/Using_in_sublime_text.md)
-
-### PHP<a name="php"></a>
-
-UPDATE: Some new options for PHP language servers are discussed in [this issue](https://github.com/tomv564/LSP/issues/259)
+* `client_name` - The client name can be any string.
+* `command` - The command line required to run the server. If not specified then `tcp_port` must be specified.
+* `tcp_port` - Sets the `tcp_port` to connect to. Else `stdin/out` is used. If no `command` is specified then it is assumed that some process is listing on this port.
+* `scopes` - Scopes of the file. Example `source.js`, `source.jsx`. To get the scopes, from the menu chose `Tools/Developer/Show Scope Name`.
+* `syntaxes` - Syntaxes enable LSP features on a document. To get the syntax name run in the sublime console: `view.settings().get("syntax")`.
+* `languageId` - Used both by the language servers and to select a syntax highlighter for sublime popups. Look up the identifiers [here](https://code.visualstudio.com/docs/languages/identifiers).
+* `enabled` - Boolean indicating if a server is enabled or disabled. You can enable a language server globally, or per-project.
+* `settings` - Per-project settings (equivalent to VS Code's Workspace Settings). Sent to server once using `workspace/didConfigurationChange` notification.
+* `env` - Dictionary of environment variables to be injected into the language server's process (eg. PYTHONPATH) Extra variables to override/add to language server's environment.
+* `initializationOptions` - Options to send to the server at startup in initialize request.
 
-#### PHP Language server
+Sometimes when a server can work with multiple type of files, you don't need to create multiple client configurations for each type. Instead you could do the following:
```

Comment:
We can avoid telling people about an option they shouldn't use here :) Just saying what the multi-language form is for should be enough.